From 3b5ec2bfab543be5a6f17cbc74a1e9f60289f885 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Tue, 14 May 2024 09:16:51 +0200
Subject: [PATCH 04/32] misc: ntsync: mark driver as "broken" to prevent from
 building

The ntsync code is only partially enabled in the kernel at this point in
time, creating the device node and that's about it.  Don't confuse
systems that expect to see a working ntsync interface by teasing it with
this basic structure at this point in time, so mark the code as "broken"
so that it is not built and enabled just yet.

Once the rest of the code is accepted, this will be reverted so that the
driver can be correctly built and used, but for now, this is the safest
way forward.

Reviewed-by: Elizabeth Figura <zfigura@codeweavers.com>
Link: https://lore.kernel.org/r/2024051450-abrasion-swizzle-550b@gregkh
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/misc/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/misc/Kconfig b/drivers/misc/Kconfig
index 801ed229e..b293e1a9e 100644
--- a/drivers/misc/Kconfig
+++ b/drivers/misc/Kconfig
@@ -508,6 +508,7 @@ config OPEN_DICE
 
 config NTSYNC
 	tristate "NT synchronization primitive emulation"
+	depends on BROKEN
 	help
 	  This module provides kernel support for emulation of Windows NT
 	  synchronization primitives. It is not a hardware driver.
-- 
2.45.1.145.g83f1add914

